app/models/run.rb

```diff
@@ -93,6 +93,7 @@ def parse(fast: true, convert: false)
         name: result[:name],
         program: result[:program],
         attempts: result[:attempts],
+        srdc_id: result[:srdc_id].blank? ? nil : result[:srdc_id],
```

Comment:
I did not realize that .present? would return false for a blank string, which was why I chose .blank?.  Will update accordingly.